src/components/common/modal/ModalActionsByFeedType.js

```diff
@@ -48,7 +49,7 @@ const ModalActionsByFeedType = ({ theme, styles, item, handleModalClose, navigat
       const { message } = exception
 
       userStorage.updateOTPLEventStatus(item.id, 'pending')
-      showErrorDialog('The payment could not be canceled at this time', code)
+      showErrorDialog('The payment could not be canceled at this time. Please try again.', code)
       log.error('cancel payment failed', message, exception, pickBy({ dialogShown: true, code, category }))
```

Comment:
```suggestion
      log.error('cancel payment failed', message, exception, pickBy({ dialogShown: true, category }))
```

Reply:
let's send exception code like in the other places (e.g. as exception.code prop)

Reply:
ok